### PR TITLE
Add tensorflow-directml-plugin to the gpu_plugins.md page

### PR DESCRIPTION
**Pull request by PatriceVignola**

### CODE REVIEW

**Review by MarkDaoust — DISMISSED** [review]
Thanks.

File: site/en/install/gpu_plugins.md [hunk 1]
```diff
@@ -60,3 +60,9 @@ Metal `PluggableDevice` for macOS GPUs:
 *   [Getting started guide](https://developer.apple.com/metal/tensorflow-plugin/){:.external}.
 *   For questions and feedback, please visit the
     [Apple Developer Forum](https://developer.apple.com/forums/tags/tensorflow-metal){:.external}.
+
+DirectML `PluggableDevice` for Windows and WSL (preview):
+
+*   [PyPI Wheel](https://pypi.org/project/tensorflow-directml-plugin/){:.external}.
+*   [GitHub Repo](https://github.com/microsoft/tensorflow-directml-plugin){.external}.
+*   For questions, feedback or to raise issues, please visit the [issues page of the github repo](https://github.com/microsoft/tensorflow-directml-plugin/issues){:.external}.
```
Comment (8bitmp3) [inline site/en/install/gpu_plugins.md:68]:
```suggestion
*   For questions, feedback or to raise issues, please visit the [Issues page of `tensorflow-directml-plugin` on GitHub](https://github.com/microsoft/tensorflow-directml-plugin/issues){:.external}.
```

File: site/en/install/gpu_plugins.md [hunk 2]
```diff
@@ -60,3 +60,9 @@ Metal `PluggableDevice` for macOS GPUs:
 *   [Getting started guide](https://developer.apple.com/metal/tensorflow-plugin/){:.external}.
 *   For questions and feedback, please visit the
     [Apple Developer Forum](https://developer.apple.com/forums/tags/tensorflow-metal){:.external}.
+
+DirectML `PluggableDevice` for Windows and WSL (preview):
+
+*   [PyPI Wheel](https://pypi.org/project/tensorflow-directml-plugin/){:.external}.
```
Comment (8bitmp3) [inline site/en/install/gpu_plugins.md:66]:
```suggestion
*   [PyPI wheel](https://pypi.org/project/tensorflow-directml-plugin/){:.external}.
```

File: site/en/install/gpu_plugins.md [hunk 3]
```diff
@@ -60,3 +60,9 @@ Metal `PluggableDevice` for macOS GPUs:
...
+*   [GitHub Repo](https://github.com/microsoft/tensorflow-directml-plugin){.external}.
```
Comment (8bitmp3) [inline site/en/install/gpu_plugins.md:67]:
```suggestion
*   [GitHub repo](https://github.com/microsoft/tensorflow-directml-plugin){.external}.
```

**Review by 8bitmp3 — DISMISSED** [review]
Thanks @PatriceVignola 

Added a few minor formatting/linting changes.

**Review by 8bitmp3 — APPROVED** [review]

File: site/en/install/gpu_plugins.md [hunk 4]
```diff
@@ -1,11 +1,11 @@
 # GPU device plugins
 
-Note: This page is for non-NVIDIA® GPU devices. For NVIDIA® GPU support, click
-[here](./gpu.md).
+Note: This page is for non-NVIDIA® GPU devices. For NVIDIA® GPU support, go to
+the [Install TensorFlow with pip](./pip) guide.
```
Comment (8bitmp3) [inline site/en/install/gpu_plugins.md:4]:
```suggestion
the [Install TensorFlow with pip](./pip.md) guide.
```

**Review by 8bitmp3 — APPROVED** [review]

**Pull request merged**